src/main/java/frc/trigon/robot/motorsimulation/ElevatorSimulation.java

```diff
@@ -0,0 +1,42 @@
+package frc.trigon.robot.motorsimulation;
+
+import edu.wpi.first.math.controller.ElevatorFeedforward;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.ElevatorSim;
+import frc.trigon.robot.utilities.Conversions;
+
+public class ElevatorSimulation extends MotorSimulation {
+    private final ElevatorSim elevatorSimulation;
+    private final double drumRadiusMeters;
+
+    public ElevatorSimulation(DCMotor motor, double gearRatio, double mass, double drumRadiusMeters, double retractedArmLengthMeters, double maximumAngleRadians, boolean simulateGravity) {
+        this.drumRadiusMeters = drumRadiusMeters;
+        elevatorSimulation = new ElevatorSim(motor, gearRatio, mass, drumRadiusMeters, retractedArmLengthMeters, maximumAngleRadians, simulateGravity, retractedArmLengthMeters);
+    }
+
+    @Override
+    double calculateFeedforward(double ks, double kg, double kv, double ka) {
+        ElevatorFeedforward feedforward = new ElevatorFeedforward(ks, kg, kv, ka);
+        return feedforward.calculate(getVelocityRevolutionsPerSecond());
+    }
```

Comment:
1. I don't want you to create the feedforward class every time. Instead, do the calculation yourself.
2. Since feedforward is open-loop is accepts only the target position / vel. 
3. Make sure you're comparing in the unit of the conversion factor.